src/controllers/organization.ts

```diff
@@ -63,24 +63,34 @@ export const getJoinOrganizationController = async (
     const { notification, siret_hint, do_not_propose_suggestions } =
       await schema.parseAsync(req.query);
 
+    let siretHint = siret_hint;
+
```

Comment:
suggestion: do not create a variable to change the case of it. I suggest you keep the original snake_case variable. snake_case is used massively in this project. I'd rather add a linter on the project to fix the case of variables that changing it here.